hits_cpp/loadSequences.cpp

```diff
@@ -16,12 +18,18 @@ SequencesMap loadSequences(string faaPath, int& numSequences, bool verbose) {
     regex pattern("^>.+_(.+)$");
     smatch matches;
     auto end = sequences.end();
-
+    int countAllSequrnces = 0;
     int count = 0;
```

Comment:
Why we need both and count and countAllSequences?
Isn't one of them (count) enough?